frontend/src/components/mods/ModOverview.tsx

```diff
@@ -0,0 +1,49 @@
+import { Checkbox, Stack, Text } from '@chakra-ui/react';
+import { useDispatch } from 'react-redux';
+import { setEnabledMod, setSelectedMod } from '../../redux/slices/mod';
+
+import ModConfig from '../../types/Configuration';
+
+const ModOverview = ({ mod, isEnabled, isSelected }: { mod: ModConfig; isEnabled: boolean; isSelected: boolean }) => {
+    const dispatch = useDispatch();
+
+    const handleEnable = () => {
+        dispatch(setEnabledMod(isEnabled ? -1 : mod.id));
+    };
+
+    const handleSelect = () => {
+        dispatch(setSelectedMod(mod.id));
+    };
+
+    return (
+        <Stack
+            direction="row"
+            px={4}
+            py={2}
+            background="gray.800"
+            borderRadius={6}
+            justifyContent="space-between"
+            alignItems="center"
+            gap={2}
+            onClick={handleSelect}
+            boxShadow={isSelected ? `0 0 0 2px var(--chakra-colors-green-300)` : undefined}
```

Comment:
hmm - any way to lift these styles out?

Reply:
There definitely will be, but it would require some changes which might be a bit of a time drain. I've added it to #21 for future reference.